packages/api/apollo-server/types/index.ts

```diff
@@ -0,0 +1,63 @@
+import { gql } from 'apollo-server-core'
+
+const Types = gql`
+  type Query {
```

Comment:
I do not recommend putting all types in the same folder. This file can get very long (hundreds of lines long). It will be hard for people to navigate through it later.

Reply:
> I do not recommend putting all types in the same folder. This file can get very long (hundreds of lines long). It will be hard for people to navigate through it later.

Why it will be that long? I thought it only contains types like `User`, `Assignment` and etc.

Reply:
What about CRUD? Do we need it for Users, Assignments, Annotations, etc?

Reply:
> What about CRUD? Do we need it for Users, Assignments, Annotations, etc?

Each operation will only take one line. e.g. `getUser: [User]`. I supposed there will be no more than 50 of them?

Reply:
Ok, in that case, let's removed the directory and name this `types.ts` - directory is redundant and confusing if we only have one file. But, I still feel that you'll find, sooner or later, that this may get pretty bloated.

Reply:
> Ok, in that case, let's removed the directory and name this `types.ts` - directory is redundant and confusing if we only have one file. But, I still feel that you'll find, sooner or later, that this may get pretty bloated.

Let's keep everything in one file for now. I heard that's how Facebook does it. We can always refactor it into smaller files once the single file becomes hard to manage.